linera-base/src/identifiers.rs

```diff
@@ -111,10 +111,7 @@ impl ChannelName {
 // Cannot use #[derive(Clone)] because it requires `A: Copy`.
 impl<A> Clone for BytecodeId<A> {
     fn clone(&self) -> Self {
-        Self {
-            message_id: self.message_id,
-            _phantom: std::marker::PhantomData,
-        }
+        *self
```

Comment:
So this is automatically `Copy` now? Does that mean `#[derive(Clone)]` would also work?

Reply:
No, it still doesn't.